apply_neighborhood improvements (#390)
Open-EO/openeo-processes: CHANGELOG.md

### Fixed

- `aggregate_spatial`:
    - Clarified that vector properties are preserved for vector data cubes and all GeoJSON Features. [#270](https://github.com/Open-EO/openeo-processes/issues/270)
    - Clarified that a `TargetDimensionExists` exception is thrown if the target dimension exists.
- `apply` and `array_apply`: Fixed broken references to the `absolute` process
- `apply_neighborhood`:

>     - Parameter `overlap` was optional but had no default value and no schena for the default value  defined.

Reviewer: ```suggestion
    - Parameter `overlap` was optional but had no default value and no schema for the default value defined.
```

>     - Clarified that the overlap must be included in the returned data cube and can't be modified. [#386](https://github.com/Open-EO/openeo-processes/issues/386)

Reviewer 2: ```suggestion
    - Clarified that the overlap must be included in the returned data cube but value changes are ignored. [#386](https://github.com/Open-EO/openeo-processes/issues/386)
```
